integration-tests/src/main/resources/application.properties

```diff
@@ -1,10 +0,0 @@
-# ------------------------------------------------------------------
-# E-MAIL
-# ------------------------------------------------------------------
-quarkus.mailer.from=
```

Comment:
I think the tests count on this `from=`?

Reply:
empty string is the quarkus default value